content/en/docs/reference/kubernetes-api/swagger/admissionregistrationv1.md

```diff
@@ -0,0 +1,7 @@
+---
+title: admissionregistration/v1
+type: swagger
+weight: 1
+---
+
+{{< swaggerui src="/docs/reference/kubernetes-api/swagger/admissionregistrationv1.json" >}}
```

Comment:
The tool/script that splits the upstream JSON needs to be checked in and maintained in the website repo, right?
When 1.20 is out, we need to regenerate all this JSON files anyway.

Reply:
These markdowns are also automatically generated, I guess. So we need that logic checked in.

Reply:
The weight value are all '1' at the moment. May need some tuning.